e2e-tests/demand-backup-cloud/run

```diff
@@ -78,12 +143,19 @@ main() {
 		cluster="demand-backup-cloud"
 		spinup_pxc "$cluster" "$test_dir/conf/$cluster.yml"
 
+		# echo "Write more data for bigger backup"
+		# run_mysql \
+		# 	'CREATE DATABASE IF NOT EXISTS sbtest;' \
+		# 	"-h $cluster-pxc -uroot -proot_password"
+		# kubectl run -it --rm sysbench-client-${RANDOM} --image=perconalab/sysbench:latest --restart=Never -- \
+		# 	sysbench oltp_read_only --tables=50 --table_size=500 --mysql-host=$cluster-pxc  --mysql-port=3306 --mysql-user=root --mysql-password=root_password --mysql-db=sbtest prepare
```

Comment:
do we want to do this?

Reply:
No, this was just for testing during draft. Removed.